js/mmoda.common.js

```diff
@@ -642,6 +642,18 @@ function get_waitingDialog($modal_dialog) {
       }
     });
 
+    //    setTimeout(function() {
+    //      //$("span.oda-icon-label").hide("slide", { direction: "right" }, 1000);
+    //      $("span.oda-icon-label").hide("slow")
+    //    }, 5000);
+
+    //    $(".main-toolbar").focusin(function() {
+    //      $("span.oda-icon-label").show("slow")
+    //    });
+    //    $(".main-toolbar").focusOut(function() {
+    //      $("span.oda-icon-label").hide("slow")
+    //    });
+
```

Comment:
I have never seen this code before, what was it used for?
Can we just remove it ?

Reply:
Done !